scripts/submit_all_script.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Loop from 1 to 10
+for i in {1..10}; do
```

Comment:
move into python source

Reply:
Don't quite understand this

Reply:
Just define this as a string directly in python rather than reading to from a file and then modifying the string